packages/ast-spec/src/token/PunctuatorToken/spec.ts

```diff
@@ -3,4 +3,18 @@ import type { BaseToken } from '../../base/BaseToken';
 
 export interface PunctuatorToken extends BaseToken {
   type: AST_TOKEN_TYPES.Punctuator;
+  value:
```

Comment:
After looking some further into the code, I discovered the `getTokenType` function in `typescript-estree`, which has 

https://github.com/typescript-eslint/typescript-eslint/blob/dac8845c6821d51cea5bafdc0be2e9a399cea68c/packages/typescript-estree/src/node-utils.ts#L519-L524

@bradzacher This means almost all values of the `TokenToText` interface (except for `InstanceOfKeyword`, `InKeyword`, `UniqueKeyword`, `KeyOfKeyword`, `NewKeyword`, `ImportKeyword` & `ReadonlyKeyword`) should be included as `value` I think?

https://github.com/typescript-eslint/typescript-eslint/blob/dac8845c6821d51cea5bafdc0be2e9a399cea68c/packages/typescript-estree/src/node-utils.ts#L16-L83

Reply:
Using all these values, has me thinking we should maybe move part of the `TokenToText` interface into a `PunctuatorTokenToText` interface and place it next to the AST spec? 🤔